CMakeLists.txt

```diff
@@ -10,9 +10,8 @@ find_package(opmonlib REQUIRED)
 
 find_package(genconfig REQUIRED)
 find_package(coredal REQUIRED)
-include_directories(${CMAKE_BINARY_DIR}/coredal ${CMAKE_BINARY_DIR}/listrev)
-daq_oks_codegen(listrev.schema.xml NAMESPACE dunedaq::listrev::dal
-  DALDIR dal DEP_PKGS coredal)
+
+daq_oks_codegen(listrev.schema.xml NAMESPACE dunedaq::listrev::dal DEP_PKGS coredal)
```

Comment:
Removing the `DALDIR` argument  breaks the compilation as the plugins expect to find the dal headers in `listrev/dal/`

Reply:
You're right. Not sure how I missed that...